dcpy/models/file.py

```diff
@@ -2,10 +2,11 @@
 from pydantic import BaseModel
```

Comment:
✂️?

Reply:
Well, `PointColumns` is technically still using it... but there's no reason that shouldn't just use sorted serialized base too. Thanks!

Reply:
oh I missed that